frontend/src/components/Util/TaskEditors/TaskEditor/index.tsx

```diff
@@ -88,9 +97,18 @@ function TaskEditor({
   memberName,
   memberEmail,
   groupID,
+  wholeTaskData,
 }: Props): ReactElement {
   const { onChange, removeTask, onSaveClicked } = actions;
+  const initTaskDate = initTaskData.date;
   const { taskData, diff, dispatchEditTask, dispatchEditSubTask, reset } = useTaskDiffReducer(
+    wholeTaskData ? ({ ...wholeTaskData, date: initTaskDate } as TaskMainData) : initTaskData,
+    wholeTaskData ? false : active ?? false,
+    wholeTaskData ? ignore : onChange ?? ignore
+  );
+
+  // only for use if the current task is a filtered completed/uncompleted task in focus view
+  const filteredTaskDiffReducerData = useTaskDiffReducer(
     initTaskData,
     active ?? false,
     onChange ?? ignore
```

Comment:
might know the reason why editing is broken now:

rendering uses filteredTaskDiffReducerData, but all the updates are applied with the functions returned by the first `filteredTaskDiffReducerData `